modules/ROOT/pages/define-and-deploy-the-bdm.adoc

```diff
@@ -152,14 +152,19 @@ When you create a process that uses a business object with a composition or aggr
 [discrete]
 ==== Loading
 
-There are two options for loading complex business objects, *lazy* or *eager*.
-With lazy loading, the parent object instance is loaded, but child object instances are loaded only when they are needed.
-With eager loading, child object instances are loaded when the parent object instance is loaded.
-
-The default is lazy loading. You can override this for any business object by configuring it to use eager loading when you specify
-the object relationship properties. This means that the specified object and all objects related to it by composition or aggregation
-relationships are loaded. +
-There is an overhead for the additional data load and maintenance of the information in memory, but there is a saving in data access time because the data is already loaded.
+There are two options for loading complex business objects, *lazy* or *eager*. +
+
+* With lazy loading, the parent object instance is loaded, and child object instances are loaded only when they are needed, by the means of additional REST API calls. This is the option to choose when the parent object along with all its relationships can get big, vertically and/or horizontally +
+* With eager loading, the child object instances are loaded when the parent object instance is loaded. It is easier to use because no additional REST API calls are needed. This is the option to choose when the parent object along with all its relationships are neither deep nor large.  +
```

Comment:
The lazy or eager choice should made based on the use case. 

Objects that are rarely changed or displayed > Lazy
Objects that are changed or display often > Eager

Even if the object is big, if it has be changed often - it will require more coding lines for the dev and in the end the performance impact will be the same as you will need to load everything that you change

Reply:
@florentsarat, can you please clarify your comment, I don't understand it. Do you want to change this documentation update?

Reply:
Yes, we need to change it. The lazy/eager choice shouldn't be decided solely based on the object size. How the object is going to be used should also be taken into account.

Reply:
Ok no problem to rephrase as long as this real warning having a critical impact on many customers gets published eventually ;-) 
I let you do the rephrase based on your offline discussions with @NathalieC, @pooravbos, Nicolas Chignardet and other. Thanks a lot!

Reply:
I'm adding Florent's comment and will merge the PR.